css/main.css

```diff
@@ -461,33 +461,144 @@ body {
   margin: 18px 0;
 }
 
-.error {
-  text-align: center;
-  color: red;
+.modal {
+  background: #fff;
+  padding: 1.5rem;
+  width: 90%;
+  opacity: 0;
+  visibility: hidden;
+  transition: transform 1s, opacity 1s, visibility 0s;
+  z-index: 300;
+}
```

Comment:
- [ ] Please from the project requirements, you should stick to the design as much as possible. But as you can see you have some differences in the design. :smiley: 
- [ ] The `project name` and the `close button` are hidden at the top of the page, hence one cannot close a modal after opening it :-1: 

| Your Design(No margin at the top) |Figma Design (There is margin at the top) |
| :---:   | :---: | 
| ![Screenshot from 2022-11-15 13-56-19](https://user-images.githubusercontent.com/92303891/201937436-cd0f0959-ea10-4a13-929b-04272287bfba.png) | ![Screenshot from 2022-11-15 13-57-46](https://user-images.githubusercontent.com/92303891/201937740-ca4194cd-dae5-46a1-881a-b08b59eff838.png) |